Meziantou.Xunit.ParallelTestFramework/EnableCustomCollectionParallelizationAttribute.cs

```diff
@@ -0,0 +1,6 @@
+namespace Meziantou.Xunit;
+
+[AttributeUsage(AttributeTargets.Class, AllowMultiple = false, Inherited = true)]
+public sealed class EnableCustomCollectionParallelizationAttribute : Attribute
```

Comment:
I think the attribute could be named `EnableParallelizationAttribute` as there is already an attribute named `DisableParallelizationAttribute`.